demo/app/ios/Runner/flutterPlugin.swift

```diff
@@ -403,20 +411,103 @@ public class SwiftWalletSDKPlugin: NSObject, FlutterPlugin {
+                                             details: "walletInitiatedOpenID4CI interaction is not initialted"))
+        }
+        
+        guard let scopesFromArgs = arguments["scopes"] as? Array<String> else {
+            return  result(FlutterError.init(code: "NATIVE_ERR",
+                                             message: "error while creating authorization Url in wallet initiated flow",
+                                             details: "Pass scopes as the arguments"))
+        }
+        
+        guard let credentialTypesFromArgs = arguments["credentialTypes"] as? Array<String> else {
+            return  result(FlutterError.init(code: "NATIVE_ERR",
+                                             message: "error while creating authorization Url in wallet initiated flow",
+                                             details: "Pass credentialTypes as the arguments"))
+        }
+        
+        guard let clientID = arguments["clientID"] as? String else {
+            return  result(FlutterError.init(code: "NATIVE_ERR",
+                                             message: "error while creating authorization Url in wallet initiated flow",
+                                             details: "Pass clientID as the arguments"))
+        }
+        
+        guard let redirectURI = arguments["redirectURI"] as? String else {
+            return  result(FlutterError.init(code: "NATIVE_ERR",
+                                             message: "error while creating authorization Url in wallet initiated flow",
+                                             details: "Pass redirectURI as the arguments"))
+        }
+        
+        
+        guard let credentialFormat = arguments["credentialFormat"] as? String else {
+            return  result(FlutterError.init(code: "NATIVE_ERR",
+                                             message: "error while creating authorization Url in wallet initiated flow",
+                                             details: "Pass credentialFormat as the arguments"))
+        }
+        
+        var scopes = ApiNewStringArray()!
+        
+        for scope in scopesFromArgs {
+            scopes.append(scope)
+        }
+        
+        
+        var credentialTypes = ApiNewStringArray()!
+        
+        for credentialType in credentialTypesFromArgs {
+            credentialTypes.append(credentialType)
+        }
+
+        do {
+            let authorizationURL = try walletInitiatedOpenID4CI.createAuthorizationURLWalletInitiatedFlow(scopes: scopes, credentialFormat: credentialFormat, credentialTypes: credentialTypes, clientID: clientID, redirectURI: redirectURI)
+            
+             result(authorizationURL)
+        } catch {
+            result(FlutterError.init(code: "NATIVE_ERR",
+                                     message: "error while creating authorization url wallet initiated issuance flow",
```

Comment:
Basically anywhere you use `url`, it should in general be capitalized since it's an acronym (some exception include at the beginning of variable names sometimes depending on coding style)